scripts/release-test-script-platform.sh

```diff
@@ -0,0 +1,242 @@
+#!/bin/bash
```

Comment:
need complete comments on each section on what your trying to achieve, so we can determine what's missing.

And create an overview of the goals for the whole file.